NPS.ID.PublicApi.DotNet.Client/Connection/Clients/StompClientFactory.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.Extensions.Logging;
+using NPS.ID.PublicApi.DotNet.Client.Connection.Enums;
+using NPS.ID.PublicApi.DotNet.Client.Connection.Options;
+using NPS.ID.PublicApi.DotNet.Client.Security.Options;
+
+namespace NPS.ID.PublicApi.DotNet.Client.Connection.Clients;
+
+public class StompClientFactory : IClientFactory
+{
+    private readonly IWebSocketConnectorFactory _webSocketConnectorFactory;
+    private readonly ILoggerFactory _loggerFactory;
+
+    private readonly TimeSpan _connectionAttemptTimeout = TimeSpan.FromSeconds(600);
```

Comment:
600 sec is a lot, does it ever make sense to wait so long?

Reply:
So what would be preferable value?

Reply:
Changed to 60